manifests/candlepin.pp

```diff
@@ -80,7 +80,6 @@
       client_cert => $client_cert,
       refreshonly => true,
       subscribe   => Exec['create-nss-db'],
-      notify      => Service['qpidd'],
```

Comment:
I am a little surprised this was needed at all. I haven't tested to verify, but it seems like qpidd shouldn't have to be restarted every time the nssdb is updated.